### DISCUSSION TITLE
One Test Run

### DISCUSSION THREAD

**Discussion by theblankist** [arkypita/LaserGRBL#968] [Q&A]
I have the Ortur Laser Master 15w laser. Just set it up. Got everything up and running. Managed to run a decent test engraving. Suddenly the laser will not work on it's lowest setting. I can get it to project the laser on a medium or high (weak) setting, but it stops after 100 seconds. When I try to run anything there is no laser projection. 

![image](https://user-images.githubusercontent.com/64119933/79939974-7389e380-842e-11ea-9825-d74a83e87964.png)
 
I have no idea what happened or how to fix this.

**Comment by theblankist** [comment]
There is no problem with stepping or moving the machine. The laser simply does not work. I ran an entire project with no problems. Tried to run it again with lower power, and now the laser won't work at all.

**Comment by arkypita** [comment]
Messages comes from your hardware, not from LaserGRBL, and they are self-explanatory.
Wait if someone know Ortur laser better then me.

**Comment by gmmanonymus111** [comment]
According to Ortur's manual, if your engraver is not in a job (or not moving) for 60 sec, or in weak mode, 100 sec, it'll turn off automatically for safety reasons. To resolve it, click on the `Lightning` icon in LaserGRBL, then the `Lock` icon.
Ortur engravers are using a different kind of microcontroller than arduino, and thus, a different version of GRBL. Since their code is not open source (at least I couldn't find it), it is impossible (very hard) to disable this feature. And since we mostly use arduino-based engravers, the compatibility of LaserGRBL with Ortur devices might be 'different'.
